module/os/freebsd/zfs/sysctl_os.c

```diff
@@ -359,19 +385,31 @@ SYSCTL_UINT(_vfs_zfs_zfetch, OID_AUTO, max_idistance, CTLFLAG_RWTUN,
 /* dsl_pool.c */
 
 /* dnode.c */
+
 extern int zfs_default_bs;
+extern int zfs_default_ibs;
+
+/* BEGIN CSTYLED */
 SYSCTL_INT(_vfs_zfs, OID_AUTO, default_bs, CTLFLAG_RWTUN,
 	&zfs_default_bs, 0, "Default dnode block shift");
-
-extern int zfs_default_ibs;
 SYSCTL_INT(_vfs_zfs, OID_AUTO, default_ibs, CTLFLAG_RWTUN,
-	&zfs_default_ibs, 0, "Default dnode indirect block shift");
-
+    &zfs_default_ibs, 0, "Default dnode indirect block shift");
+/* END CSTYLED */
 
 /* dsl_scan.c */
 
 /* metaslab.c */
 
+extern int zfs_metaslab_sm_blksz_no_log;
+extern int zfs_metaslab_sm_blksz_with_log;
+extern int zfs_condense_pct;
+extern int zfs_remove_max_segment;
+extern int zfs_removal_suspend_progress;
+extern uint64_t metaslab_df_alloc_threshold;
+extern int metaslab_df_free_pct;
+extern int metaslab_load_pct;
+extern int metaslab_preload_limit;
+
```

Comment:
I suppose there is some good formal reason to organize it this way, but I'd personally prefer everything related to one parameter/variable to be in one place.

Reply:
I split it up and even found one missing legacy sysctl for `l2arc_headroom_boost`.